### PR TITLE
build: Drop --enable-old-bridge configure option

### PR DESCRIPTION
**Pull request by martinpitt**
This isn't supported on the main branch any more since we dropped supprot for RHEL 8. This was already cleaned up in the spec file and tests. `HACK_SPEC_FOR_PYTHON` is also not a thing any more.

Also clean up the four ws tests which we haven't run any more since then.

Much of src/bridge can also go now, but we'll leave that for #20000. As automake requires a defined conditional for `if`, define an always-false `WITH_OLD_BRIDGE` for now.

----

NB: This is *not* the big cleanup from #20000 yet. I added that conditional hack specifically to avoid that :grin:  A lot of that can happen now, but @allisonkarlitskaya reserved that fun for her. I mostly just want to get rid of the OLD_BRIDGE configure stuff as I keep running into it while working on #19441

### CODE REVIEW

File: src/bridge/Makefile.am [hunk 1]
```diff
@@ -39,6 +39,7 @@ libcockpit_metrics_a_SOURCES = \
 	src/bridge/cockpitsamples.h \
 	$(NULL)
 
+# TODO: remove all of the below
 if WITH_OLD_BRIDGE
```
Comment (martinpitt) [inline src/bridge/Makefile.am:43]: FTR: I tried with `if 0` or `if REMOVE_ME` etc., but none of that works. There just isn't a multi-line "disable this section" in automake that I know of :cry: I am happy to just remove the block here, but then we should also remove all the files (as this Makefile block is the reference list of which files  can go), and then I step too much on Lis'es toes.

**Comment by martinpitt** [comment]
@allisonkarlitskaya: This doesn't block me after all, so I'll leave it up to you what you want to do with this -- land, reject, or expand into #20000.

**Comment by martinpitt** [comment]
@jelly yes, it is -- this does *not* drop cockpit-pcp, nor does it drop the cockpit-bridge code. See the details in the PR description. This only drops the `--enable-old-bridge` config option, which we really should have done months ago.

**Comment by jelly** [comment]
> @jelly yes, it is -- this does _not_ drop cockpit-pcp, nor does it drop the cockpit-bridge code. See the details in the PR description. This only drops the `--enable-old-bridge` config option, which we really should have done months ago.

Right, but can I still then build `cockpit-pcp` easily after this?

**Comment by martinpitt** [comment]
@jelly yes, both cockpit-pcp and cockpit-ssh are still built from C sources today, they don't count as "old". This only applies to `cockpit-bridge` itself.

**Review by jelly — APPROVED** [review]

**Review by allisonkarlitskaya — APPROVED** [review]
Yes.  This no longer makes sense, and we've already made some changes that work towards disabling support for running Qunit tests on the old bridge, so this is probably already kinda broken anyway...

**Pull request merged**